lib/cli/src/Cardano/CLI.hs

```diff
@@ -262,8 +262,6 @@ runCli = join . customExecParser preferences
 
 {-------------------------------------------------------------------------------
                             Commands - 'mnemonic'
-
-  cardano-wallet mnemonic generate [--size=INT]
```

Comment:
Do you plan to also remove the comments on functions like the one below:
```hs
-- | cardano-wallet mnemonic generate [--size=INT]
cmdMnemonicGenerate :: Mod CommandFields (IO ())         
```
It seems these are now superfluous, as we have:
```hs
["mnemonic", "generate", "--help"] `shouldShowUsage`  
    [ "Usage:  mnemonic generate [--size INT]"
    , "  Generate English BIP-0039 compatible mnemonic words."

```